contents/tutorials/redirect-testing.md

```diff
@@ -0,0 +1,383 @@
+---
+title: How to do redirect testing
+date: 2023-10-19
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-16.png
+tags: ['experimentation', 'feature flags']
+---
+
+Redirect testing is a way to A/B test web pages by redirecting users to one or the other. It is a popular way to split test landing pages, for example.
+
+To show you how to set up a redirect test with PostHog, we create a two-page Next.js app, create an experiment in PostHog, and then implement it in our app using middleware and feature flags. 
```

Comment:
Let's stick with A/B test, since thats what we refered to it above
```suggestion
To show you how to set up a redirect test with PostHog, we create a two-page Next.js app, create an A/B test in PostHog, and then implement it in our app using middleware and feature flags. 
```